app/package.json

```diff
@@ -21,12 +21,12 @@
   "dependencies": {
     "@trussworks/react-uswds": "^4.1.1",
     "@uswds/uswds": "3.1.0",
-    "i18next": "^22.4.12",
+    "i18next": "^23.0.0",
```

Comment:
From the Changelog, which we may want to update:

> [i18next 23.0.0](https://togithub.com/i18next/i18next/releases/tag/v23.0.0) dropped support for older browsers.
From nextjs 13, you can use the [transpilePackages](https://nextjs.org/docs/app/api-reference/next-config-js/transpilePackages)
to avoid issues.